common/src/main/java/org/apache/celeborn/common/network/server/memory/ReadBufferDispatcher.java

```diff
@@ -63,24 +69,26 @@ public void run() {
     while (true) {
       ReadBufferRequest request = null;
       try {
-        request = requests.poll(500, TimeUnit.MILLISECONDS);
+        request = requests.poll(1000, TimeUnit.MILLISECONDS);
       } catch (InterruptedException e) {
         logger.info("Buffer dispatcher is closing");
       }
 
       if (request != null) {
+        long start = System.nanoTime();
         List<ByteBuf> buffers = new ArrayList<>();
         int bufferSize = request.getBufferSize();
         while (buffers.size() < request.getMin()) {
           if (memoryManager.readBufferAvailable(bufferSize)) {
             memoryManager.changeReadBufferCounter(bufferSize);
             ByteBuf buf = readBufferAllocator.buffer(bufferSize, bufferSize);
             buffers.add(buf);
+            allocatedReadBuffers.increment();
```

Comment:
Will allocatedReadBuffers overflow? Since it keeps incrementing.

Reply:
It will decrement when a buffer is recycled.